showcases/patternhub/pages/foundations/icons/overview.tsx

```diff
@@ -9,17 +9,52 @@ import {
 } from '../../../../../output/react/src';
 import DefaultPage from '../../../components/default-page';
 
+// Import root package.json for theme version
+import rootPackage from '../../../../../package.json';
+// Import the theme-icons version constant
+import themeIconsPackage from '../../../../../node_modules/@db-ux/db-theme-icons/package.json';
+
 const IconOverview = () => {
 	const [weight, setWeight] = useState<string>('24');
 	const [family, setFamily] = useState<string>('default');
 	// TODO: we should add a better search for this
 	const [search, setSearch] = useState<string>('');
+
+	// Get theme version from root package.json
+	const themeVersion =
+		rootPackage.devDependencies?.['@db-ux/db-theme'] ??
+		rootPackage.dependencies?.['@db-ux/db-theme'] ??
+		'unknown';
+	const themeIconsVersion = themeIconsPackage.version ?? 'unknown';
 	return (
 		<DefaultPage>
 			<h1>Icon overview</h1>
 			<p>
 				We don't provide all icons with family <code>filled</code>
 			</p>
+			<p>
+				These icons reflect the
+				<a
+					href={`https://www.npmjs.com/package/@db-ux/db-theme-icons/v/${themeIconsVersion}`}>
+					<code>@db-ux/db-theme-icons</code> node package
+				</a>{' '}
+				of version{' '}
+				<a
+					href={`https://www.npmjs.com/package/@db-ux/db-theme-icons/v/${themeIconsVersion}`}>
```

Comment:
```suggestion
					href={`https://www.npmjs.com/package/@db-ux/db-theme-icons/v/${themeIconsVersion}`}
					target="_blank" rel="noopener noreferrer">
```
I would open the link in a new window